src/main/kotlin/com/glycin/ragvillage/ai/CommandJudge.kt

```diff
@@ -0,0 +1,23 @@
+package com.glycin.ragvillage.ai
+
+import com.glycin.ragvillage.model.VillagerCommand
+import dev.langchain4j.service.SystemMessage
+import dev.langchain4j.service.UserMessage
+import io.quarkiverse.langchain4j.RegisterAiService
+
+@RegisterAiService(modelName = "chat", chatMemoryProviderSupplier = LimitedChatMemory::class)
+interface CommandJudge {
+    @SystemMessage("""
+                You are a validation agent tasked with verifying the responses generated by a primary agent for a role-playing game set in Little Minas Morgul. 
+                Your role is to extract structured data from the input you receive.
+                                
+                Your output must STRICTLY (I MEAN IT!) ADHERE TO THE FOLLOWING JSON FORMAT
```

Comment:
If it happens sometimes that the LLM doesn't respond correctly, consider adding guardrails - https://docs.quarkiverse.io/quarkus-langchain4j/dev/guardrails.html